opam

```diff
@@ -5,9 +5,12 @@ homepage: "https://github.com/ocamllabs/higher"
 bug-reports: "https://github.com/ocamllabs/higher/issues"
 dev-repo: "https://github.com/ocamllabs/higher.git"
 license: "MIT"
-build: [
-  "ocaml" "pkg/build.ml" "native=%{ocaml-native}%"
-                         "native-dynlink=%{ocaml-native-dynlink}%"
+build:
+[[ "ocaml" "pkg/pkg.ml" "build"
+           "--installer" "true" ]]
```

Comment:
Formally speaking you could eschew the `"--pinned" "%{pinned}%"` in the OPAM repo since the one of the git repo will be read and will take over when pinned, but in practice you want to deal with a single `opam` in all situations (and `topkg`'s OPAM release process will simply take the one that is in your repo).